components/Cards/src/MDCCard.m

```diff
@@ -43,6 +44,7 @@ @implementation MDCCard {
   NSMutableDictionary<NSNumber *, UIColor *> *_borderColors;
   UIColor *_backgroundColor;
   CGPoint _lastTouch;
+  BOOL _isInteractable;
```

Comment:
super-nit:
s/_isInteractable/interactable/
Only the getter is prefixed with is.
I would expect this to be auto-synthesized.